src/AppBanner/index.tsx

```diff
@@ -39,7 +39,8 @@ export const AppBanner = (props: Props): ReactElement | null => {
         trackClick,
     } = props;
 
-    const styles = selfServeStyles(props.brazeMessageProps, defaultColors);
+    const bannerWithLinkBaseColors = getColors(props.brazeMessageProps, defaultColors);
```

Comment:
For this banner I don't think `props.brazeMessageProps` ever contains colour props. I wonder if it would be clearer to pass an empty object `{}`? Or maybe even better update `getColors` to make the override colour props optional?

Reply:
Updated